### PR TITLE
Feature/120 update docs based on new devex features in zilla examples

### PR DESCRIPTION
**Pull request by vordimous**

### CODE REVIEW

File: src/tutorials/sse/echo/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  sse-server:
+    image: zilla-examples/sse-server:local
+    hostname: sse-server
+    tty: true
+    ports:
+      - "8001:8001"
+      - "7001:7001"
+    command: -v -p 8001 -i 7001
+
+  sse_messenger:
```
Comment (jfallows) [inline src/tutorials/sse/echo/docker-compose.yaml:26]:
```suggestion
  sse_publisher:
```

File: src/tutorials/sse/echo/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```
Comment (jfallows) [inline src/tutorials/sse/echo/index.html:1]: `echo` has a meaning of reflecting back what was sent, but this is an `sse` `proxy` example, not `sse` `echo`.
Suggest we rename directory structure to `sse/proxy`.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: src/concepts/config-intro.md [hunk 3]
```diff
@@ -139,7 +139,7 @@ bindings:
   tls_server:
     type: tls
     kind: server
-    vault: your_servers
+    vault: my_servers
```
Comment (jfallows) [inline src/concepts/config-intro.md:142]: Looks good. 👍 

Btw, several other config files need to also change from `your_servers` to `my_servers` for consistency.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: src/concepts/config-intro.md (hunk 3 above)
Comment (vordimous) [reply src/concepts/config-intro.md:142]: right, thank you!

**Review by jfallows — DISMISSED** [review]

**Review by jfallows — APPROVED** [review]

**Pull request merged**